R/helpers.R

```diff
@@ -510,3 +510,27 @@ myBarplot <- function(x, description, xaxisround=0)
                          round(dat$x, xaxisround))
 }
 
+#' Wrapper for all.equal
```

Comment:
hm ok I see - can you double check if `expect_equivalent` does not already do something very similar maybe? now that I see this I remember that function

Reply:
`expect_equivalent ` seems to be deprecated by `expect_equal`. And `expect_equal` suffers from the same problems as `all.equal` which we solved using by the wrapper `all_equal`.